src/main/java/teamfour/tasc/commons/events/ui/CollapseChangeEvent.java

```diff
@@ -0,0 +1,26 @@
+package teamfour.tasc.commons.events.ui;
+
+import teamfour.tasc.commons.events.BaseEvent;
+
+public class CollapseChangeEvent extends BaseEvent {
+
+    private static boolean isCollapsed = false;
+
+    public CollapseChangeEvent(boolean toCollapse){
```

Comment:
Should put a comment to tell people that all objects and constructors of this class sets and shares a static variable